examples/state/src/request_local.rs

```diff
@@ -1,6 +1,6 @@
 use std::sync::atomic::{AtomicUsize, Ordering};
 
-use rocket::State;
+use rocket::{State, StateMissing};
```

Comment:
Should probably be named `StateError` or something? Looks to be what the stdlib prefers.